### PR TITLE
v4/ Marketplace coming soon

### PR DESCRIPTION
**Pull request by ronronscelestes**
## What

Integrated new page layout for marketplace coming soon view
⚠️ Submit your plugin link temporarily commented waiting for the right URL

## Snapshot 

Screenshot 2021-11-09 at 16 30 42

### CODE REVIEW

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/core/admin/admin/src/pages/MarketplacePage/index.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ import {
   // useAppInfos,
   // useTracking,
   // useStrapiApp,
+  pxToRem,
```
Comment (soupette) [inline packages/core/admin/admin/src/pages/MarketplacePage/index.js:10]: Can you remove all the comments?

File: packages/core/admin/admin/src/translations/en.json [hunk 2]
```diff
@@ -266,7 +266,14 @@
   "app.components.InstallPluginPage.Download.title": "Downloading...",
   "app.components.InstallPluginPage.description": "Extend your app effortlessly.",
   "app.components.InstallPluginPage.helmet": "Marketplace - Plugins",
-  "app.components.InstallPluginPage.title": "Marketplace - Plugins",
+  "app.components.InstallPluginPage.illustration": "marketplace illustration",
```
Comment (soupette) [inline packages/core/admin/admin/src/translations/en.json:269]: The translations should start by the following: `admin.pages.MarketPlacePage.`

**Review by soupette — CHANGES_REQUESTED** [review]
Can you add a test like a snapshot?

File: packages/core/admin/admin/src/pages/MarketplacePage/index.js [hunk 3]
```diff
@@ -116,9 +116,29 @@ import {
...
+import { Helmet } from 'react-helmet';
 import { Layout, HeaderLayout, ContentLayout } from '@strapi/design-system/Layout';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+import { LinkButton } from '@strapi/design-system/LinkButton';
 import { Main } from '@strapi/design-system/Main';
+import { Typography } from '@strapi/design-system/Typography';
 import adminPermissions from '../../permissions';
+import MarketplacePicture from '../../assets/images/marketplace-coming-soon.png';
```
Comment (soupette) [inline packages/core/admin/admin/src/pages/MarketplacePage/index.js:129]: Can you put the png file inside the MarketPlace folder?

`admin/admin/src/pages/MarketplacePage/assets/marketplace-coming-soon.png`

**Review by soupette — APPROVED** [review]
LGTM

**Pull request merged**